specification/storagepool/resource-manager/Microsoft.StoragePool/stable/2021-08-01/examples/IscsiTargets_Get.json

```diff
@@ -0,0 +1,59 @@
+{
+  "parameters": {
+    "api-version": "2021-08-01",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "myResourceGroup",
+    "diskPoolName": "myDiskPool",
+    "iscsiTargetName": "myIscsiTarget"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myResourceGroup/providers/Microsoft.StoragePool/diskPools/myDiskPool/iscsiTargets/myIscsiTarget",
+        "name": "myIscsiTarget",
+        "type": "Microsoft.StoragePool/diskPools/iscsiTargets",
+        "systemData": {
+          "createdAt": "2020-06-24T06:53:57+00:00",
+          "createdBy": "alias",
+          "createdByType": "User",
+          "lastModifiedAt": "2020-06-24T06:53:57+00:00",
+          "lastModifiedBy": "alias",
+          "lastModifiedByType": "User"
+        },
+        "properties": {
+          "targetIqn": "iqn.2005-03.org.iscsi:server1",
+          "aclMode": "Static",
+          "luns": [
+            {
+              "managedDiskAzureResourceId": "/subscriptions/11111111-1111-1111-1111-111111111111/resourceGroups/myResourceGroup/providers/Microsoft.Compute/disks/vm-name_DataDisk_1",
+              "name": "lun0",
+              "lun": 0
+            }
+          ],
+          "staticAcls": [
+            {
+              "initiatorIqn": "iqn.2005-03.org.iscsi:client",
+              "mappedLuns": [
+                "lun0"
+              ]
+            }
+          ],
+          "endpoints": [
+            "10.0.0.1:3260"
+          ],
+          "activeSessions": [
+            "iqn.2005-03.org.iscsi:client"
+          ],
+          "port": 3260,
+          "provisioningState": "Succeeded",
+          "status": "Healthy"
+        }
+      },
+      "managedBy": "/subscriptions/11111111-1111-1111-1111-111111111111/resourceGroups/myResourceGroup/providers/Microsoft.AVS/privateClouds/myPrivateCloud/clusters/Cluster-1/datastores/datastore1",
+      "managedByExtended": [
```

Comment:
@jorgecotillo Can you also confirm that it is ok to set `managedBy `and `managedBy `extended on our tracked resource (DiskPool) and our proxy/child resource (IscsiTarget).

Reply:
I'll get back to you on this one.

Reply:
Ok, I got answers, yes it is ok to set on both tracked and proxy resources.

Reply:
Thanks!